pkg/minikube/node/start.go

```diff
@@ -156,11 +156,13 @@ func Start(starter Starter, apiServer bool) (*kubeconfig.Settings, error) {
 			return nil, errors.Wrap(err, "Updating node")
 		}
 
-		cpBs, err := cluster.Bootstrapper(starter.MachineAPI, viper.GetString(cmdcfg.Bootstrapper), *starter.Cfg, starter.Runner)
+		// Load the control plane host
+		c := mustload.Running(starter.Cfg.Name)
+
+		cpBs, err := cluster.Bootstrapper(starter.MachineAPI, viper.GetString(cmdcfg.Bootstrapper), *starter.Cfg, c.CP.Runner)
 		if err != nil {
-			return nil, errors.Wrap(err, "Getting bootstrapper")
+			return nil, errors.Wrap(err, "Failed to get control plane bootstrapper")
```

Comment:
to match other error.Wraps, it should be, errors.Wrapf(err, "getting bootstrapper for %s",starter.Cfg.Name)

Reply:
add the name of the cluster doesn't really help since there's no confusion about what cluster you're trying to start, I did change the format of the error message to conform with the rest of the file